protobufs/livekit_models.proto

```diff
@@ -282,6 +284,15 @@ message TranscriptionSegment {
   string language = 6;
 }
 
+message TimeSyncRequest {
+  string id = 1;
+}
+
+message TimeSyncResponse {
+  string id = 1;
+  uint64 ntp_time = 2;
```

Comment:
we've talked about ways to better distinguish request and response timings of the RTT from the client's perspective. You mentioned the NTP protocol as an example of that. Are we ditching that idea?

Reply:
NTP distinguishes between the time the request got into the the server, and the time the response leaves the server. In this case, since the handler does nothing else than get the current time and write it into the response, these 2 times would be the same. We can add an extra field in the future if extra processing is needed on the server. 

Either way, we are not able to distinguish between the time spent in either part of the round trip of the synchronization call. 

The wikipedia article on this topic mentions:

```
Accurate synchronization is achieved when both the incoming and outgoing routes between the client and the server have symmetrical nominal delay. If the routes do not have a common nominal delay, a [systematic bias](https://en.wikipedia.org/wiki/Systematic_bias) exists of half the difference between the forward and backward travel times. A number of approaches have been proposed to measure asymmetry,[[39]](https://en.wikipedia.org/wiki/Network_Time_Protocol#cite_note-iL6pp-41) but among practical implementations only chrony seems to have one included.[[40]](https://en.wikipedia.org/wiki/Network_Time_Protocol#cite_note-Ocilw-42)[[41]](https://en.wikipedia.org/wiki/Network_Time_Protocol#cite_note-43)
```

Reply:
makes sense, thanks for the explanation!